templates/evidence.template

```diff
@@ -0,0 +1,20 @@
+#[Title]#
+%evidence.description%
+
+#[File]#
+%evidence.file%
+
+#[Line]#
+%evidence.line%
+
+#[Column]#
+%evidence.column%
+
+#[Method]#
+%evidence.method%
```

Comment:
Should the :context, the :caller and the :method be inside `bc` blocks?

In one of the samples I have, I got this Context:

`_call . org.apache.axis.client.Call.invoke ( new java.lang.Object[][1] )`

Which contains Textile-meaningful syntax, so it rendered as:

<img width="473" alt="appscan-02" src="https://user-images.githubusercontent.com/53006/27907786-533a8f9e-6249-11e7-894d-a3f80688890f.png">